### PR TITLE
KV Cache, E2E Tests

### PR DESCRIPTION
**Pull request by horheynm**
SUMMARY:
Create e2e tests for using KVCache

### CODE REVIEW

**Review by dsikka — APPROVED** [review]
Very nice!

**Review by kylesayrs — APPROVED** [review]

**Review by kylesayrs — APPROVED** [review]
Should run with nightly runners before merging

**Comment by markurtz** [comment]
@kylesayrs has this run with nightly and can we get it landed?

**Review by dsikka — COMMENTED** [review]
Can this land? @horheynm

File: tests/e2e/vLLM/run_tests.sh [hunk 1]
```diff
@@ -5,26 +5,30 @@ SUCCESS=0
...
     echo "=== RUNNING MODEL: $MODEL_CONFIG ==="
-
-    export TEST_DATA_FILE="$MODEL_CONFIG"
-    pytest \
-        --capture=tee-sys \
-        --junitxml="test-results/e2e-$(date +%s).xml" \
-        "$PWD/tests/e2e/vLLM/test_vllm.py" || LOCAL_SUCCESS=$?
-
+    export TEST_DATA_FILE=${MODEL_CONFIG}
+    pytest -s $PWD/tests/e2e/vLLM/test_vllm.py || LOCAL_SUCCESS=$?
```
Comment (dbarbuzzi) [inline tests/e2e/vLLM/run_tests.sh:15]: This change to the `pytest` command is unnecessary and removes important, required flags that were recently added.

Comment (horheynm) [reply tests/e2e/vLLM/run_tests.sh:15]: Ok

**Review by dsikka — CHANGES_REQUESTED** [review]
Are any of the configs using the gptq recipe?
Have we confirmed all the configs work?

**Comment by horheynm** [comment]
@dsikka its ready

**Review by dsikka — APPROVED** [review]

**Pull request merged**